### ISSUE TITLE
iron-swipeable-container doesn't work on polymer 2.0 or hybrid projects

### ISSUE THREAD

**Issue by ernysans** [PolymerElements/iron-swipeable-container#27]
### Description
`iron-swipeable-container` doesn't show the slotted content using polymer 2.0.

### Expected outcome
`iron-swipeable-container` should show the slotted content

### Actual outcome
Don't displays the slotted content.

### Live Demo
Standard demo included in the element: https://github.com/PolymerElements/iron-swipeable-container/blob/master/demo/index.html

#### Option 1:
1. Update standard demo to Polymer 2.0
2. Run polymer serve
3. Try to swipe an element
#### Option 2:
1. Include `iron-swipeable-container` in a Polymer 2 project
2. Try to swipe an element

### Browsers Affected
- [x] Chrome
- [x] Firefox
- [x] Safari 9
- [x] Safari 8
- [x] Safari 7
- [x] Edge
- [x] IE 11
- [x] IE 10

**Comment by Mikebarson** [comment]
@ernysans  Has this been fixed?

**Comment by ralphsmith80** [comment]
There was an issue #25 with a fix that looks to solve this in my testing. I installed it from ` Collaborne/iron-swipeable-container` to get things working. Looks like it just waiting on a merge into the main branch.
